impl/pom.xml

```diff
@@ -54,6 +54,11 @@
             <artifactId>jjwt-orgjson</artifactId>
             <scope>test</scope>
         </dependency>
+	<dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-gson</artifactId>
+	   <version>${project.version}</version>
```

Comment:
Not quite.  Drop the version tag, and add a test scope like the one above it, and make sure the formatting is correct.

Reply:
Excuse me i ude gradle basically. Whats wrong with farmatting? do i need to use tabs?

Reply:
You have less spaces than the line above it. so it is misaligned.